R/cluster.R

```diff
@@ -84,7 +84,8 @@ generateClusterConfig <- function(fileName, ...){
       ),
       rPackages = list(
         cran = vector(),
-        github = vector()
+        github = vector(),
+        authToken = NULL
```

Comment:
Is the authToken only for gitHub? If so, I would rename to githubAuthToken or consider making this a complex object (probably not ideal for backwards compatibility).

@brnleehng - thoughts on this?

Also, we probably should update some docs to help people understand this.